libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1418,6 +1418,16 @@ void ExpressionCompiler::appendArithmeticOperatorCode(Token::Value _operator, Ty
 		m_context << Instruction::DUP2 << Instruction::ISZERO;
 		m_context.appendConditionalInvalid();
 
+		// Test for (minInt) / (-1)
+		if (c_isSigned && _operator == Token::Div)
+		{
+			m_context.appendInlineAssembly(R"(
+				and(eq(x, u256(1) << 255), eq(y, -1))
```

Comment:
I don't think `u256(1) << 255` works?

It has to be a supplied constant in this case or just hardcode the value.